LEAF_Request_Portal/form.php

```diff
@@ -526,13 +526,15 @@ public function getIndicatorLog($indicatorID, $series, $recordID)
                       ':indicatorID' => $indicatorID,
                       ':series' => $series, );
 
+
         $res = $this->db->prepared_query(
-            'SELECT * FROM data_history
-                LEFT JOIN indicator_mask USING (indicatorID)
-                WHERE recordID=:recordID
-                AND indicatorID=:indicatorID
-                AND series=:series
-                ORDER BY timestamp DESC', 
+            'SELECT h.recordID, h.indicatorID, h.series, h.data, h.timestamp, h.userID, i.is_sensitive 
+                FROM data_history h, indicators i 
+                    LEFT JOIN indicator_mask USING (indicatorID)
+                    WHERE h.recordID=:recordID
+                    AND h.indicatorID=:indicatorID
+                    AND h.series=:series
+                    ORDER BY timestamp DESC',
             $vars
```

Comment:
Looks like I made a mistake in my first comment 🤷‍♂️  The query should read:

```sql
SELECT h.recordID, h.indicatorID, h.series, h.data, h.timestamp, h.userID, i.is_sensitive 
                FROM data_history h
                    LEFT JOIN indicator_mask USING (indicatorID)
                    LEFT JOIN indicators i USING (indicatorID)
                    WHERE h.recordID=:recordID
                    AND h.indicatorID=:indicatorID
                    AND h.series=:series
                    ORDER BY timestamp DESC
```

As it is now, when doing a SELECT FROM on `indicators`, it returns multiple rows of the same item from {{data_history}} which results in the indicator log having multiple entries for the same bit of history.